lib/auth/bloc/auth_bloc.dart

```diff
@@ -22,36 +23,35 @@ class AuthBloc extends Bloc<AuthEvent, AuthState> {
     AuthEvent event,
   ) async* {
     if (event is AuthSignedIn) {
-      final String token = await authRepositories.signIn(event.body);
-      if (token != null) {
+      final response = await authRepositories.signIn(event.body);
```

Comment:
I think you need extract event handlers to own methods to make the code more readable.

Reply:
It refers to all blocs.

Reply:
Fix